Code/Tools/AssetProcessor/native/ui/style/AssetProcessor.qss

```diff
@@ -155,3 +156,29 @@ QListWidget::item:hover {
 QListWidget QHeaderView::section {
     background-color: rgb(34,34,34);
 }
+
+QLabel#sharedCacheStatus {
+    background-color: transparent;
+    color : white;
+}
+
+QLabel#sharedCacheStatus[highlight = "blue"] {
```

Comment:
This is really a nitpick, you might want to consider more meaningful names (error, normal, etc) in case the colors ever change to not match the name